body.tex

```diff
@@ -128,15 +132,16 @@ \subsection{The Tools Used}
 \paragraph{Docushare}
 is the official Rubin Observatory documentation repository during construction.
 Documents are uploaded in Docushare only after their formal approval.
+%by the CCB ? But other things go in docushare wihtout approval ..
```

Comment:
Since the sentence is not true, I remove it. 
When I say formal approval, I am not only referring to a CCB approval, but also to formal test report approved by the responsible of the test milestone.